pxtblocks/blocklyloader.ts

```diff
@@ -913,21 +1058,168 @@ namespace pxt.blocks {
             return xmlList;
         };
 
+        // builtin variables_get
+        let msg:any = Blockly.Msg;
+        msg.VARIABLES_GET_CREATE_SET = lf("Create 'set %1'");
+        installHelpResources(
+            'variables_get',
+            lf("get the value of a variable"),
+            lf("Returns the value of this variable."),
+            '/blocks/variables'
+        );
+
+        // builtin variables_set
+        msg.VARIABLES_SET = lf("set %1 to %2");
+        msg.VARIABLES_DEFAULT_NAME = lf("_default_var");
```

Comment:
Same as ``_for_do`` and ``_while_do``, please open an issue and we'll fix this in a future commit.